certbot/plugins/selection.py

```diff
@@ -108,11 +108,19 @@ def choose_plugin(prepared, question):
     opts = [plugin_ep.description_with_name +
             (" [Misconfigured]" if plugin_ep.misconfigured else "")
             for plugin_ep in prepared]
+    names = set(plugin_ep.name for plugin_ep in prepared)
 
     while True:
         disp = z_util(interfaces.IDisplay)
-        code, index = disp.menu(
-            question, opts, force_interactive=True)
+        if "CERTBOT_AUTO" in os.environ and names == set(("apache", "nginx")):
```

Comment:
It may be slightly better to use `certbot.cli.cli_command == certbot.cli.LEAUTO`, in case we're somehow in a really, really old non-updating `letsencrypt-auto` venv?